test/unit/auth/test_jwt_auth.py

```diff
@@ -364,3 +364,99 @@ def test_refresh_instance_sends_post_request_with_correct_params(jwt_auth_init_a
     enterprise_id = 'fake_enterprise_id'
     with jwt_auth_init_and_auth_mocks(enterprise_id, 'enterprise', enterprise_id=enterprise_id) as oauth:
         oauth.refresh(None)
+
+
+@pytest.fixture()
+def jwt_subclass_that_just_stores_params():
+    class StoreParamJWTAuth(JWTAuth):
+        def __init__(self, **kwargs):
+            self.kwargs = kwargs
+
+    return StoreParamJWTAuth
+
+
+@pytest.fixture
+def fake_client_id():
+    return 'fake_client_id'
+
+
+@pytest.fixture
+def fake_client_secret():
+    return 'fake_client_secret'
+
+
+@pytest.fixture
+def fake_enterprise_id():
+    return 'fake_enterprise_id'
+
+
+@pytest.fixture
+def app_config_json_content(
+        fake_client_id,
+        fake_client_secret,
+        fake_enterprise_id,
+        jwt_key_id,
+        rsa_private_key_bytes,
+        rsa_passphrase,
+):
+    return r"""
+{{
+  "boxAppSettings": {{
+    "clientID": "{client_id}",
+    "clientSecret": "{client_secret}",
+    "appAuth": {{
+      "publicKeyID": "{jwt_key_id}",
+      "privateKey": "{private_key}",
+      "passphrase": "{passphrase}"
```

Comment:
This isn't quite right, because:

If the `rsa_passphrase` fixture is ever `None`, it will get written as `"None"` instead of `""` or `null`.

Try `passphrase=(rsa_passphrase or '')` in the `.format()` call.

Similarly for `fake_enterprise_id`.